.semaphore/semaphore.yml

```diff
@@ -0,0 +1,28 @@
+version: v1.0
+name: Mainflux IoT platform
+agent:
+  machine:
+    type: e1-standard-4
+    os_image: ubuntu1804
+
+blocks:
+  - name: Build
+    task:
+      jobs:
+        - name: Build and test
+          commands:
+            - export GOPATH=~/go
+            - export GOBIN=$GOPATH/bin
+            - export PATH=$PATH:$GOPATH:$GOBIN
+            - PROTOC_ZIP=protoc-3.6.1-linux-x86_64.zip
+            - curl -0L https://github.com/google/protobuf/releases/download/v3.6.1/$PROTOC_ZIP -o $PROTOC_ZIP
+            - unzip -o $PROTOC_ZIP -d protoc3
+            - sudo mv protoc3/bin/* /usr/local/bin/
+            - sudo mv protoc3/include/* /usr/local/include/
+            - rm -f PROTOC_ZIP
+            - export PATH=$PATH:/usr/local/bin/protoc
+            - go get -u github.com/golang/protobuf/protoc-gen-go github.com/gogo/protobuf/protoc-gen-gofast google.golang.org/grpc
+            - go get -d github.com/mainflux/mainflux
+            - cd $GOPATH/src/github.com/mainflux/mainflux
+            - make proto
+            - set -e; echo "" > coverage.txt; for d in $(go list ./... | grep -v 'vendor\|cmd'); do GOCACHE=off go test -v -race -tags test -coverprofile=profile.out -covermode=atomic $d; if [ -f profile.out ]; then cat profile.out >> coverage.txt; rm profile.out; fi done
```

Comment:
I'd suggest use of multiline string (Block Scalar >) as described [here](https://stackoverflow.com/a/21699210) for better visibility, but I'm not sure if it will work with Semaphore CI. Other than that all looks good to me

Reply:
@chombium We've decided not to switch to Semaphore 2 yet because it's still in Beta phase. This PR will add simple shell script that will be executed by Semaphore so that we can track CI commands changes.